src/lib/Scenes/City/Components/CityTab.tsx

```diff
@@ -31,6 +31,7 @@ export class CityTab extends React.Component<Props> {
         keyExtractor={item => item.node.id}
         renderItem={({ item }) => this.renderItem(item)}
         scrollEnabled={false}
+        windowSize={50}
```

Comment:
An hour of faffing with Xcode and all I got was this 1 line change

Reply:
bah

Reply:
To be fair, that was mostly just me catching up with the status quo in general and specifically on my iMac

Reply:
hab

Reply:
wow, one liner fixes 😩